scripts/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+## load variables from the env file
+source vars
+
+create_base_dir(){
+    mkdir -p ${Base_Dir}
+}
+
+create_data_dir(){
+  mkdir -p ../data
```

Comment:
@jmbabazi can you explain how the base dir and data dir are related to each other?  Is the data dir not a subdirectory of the base dir?

Reply:
I made some few changes @mseaton . The idea is to have a Data_Dir which is the folder for MySQL on the host. In this folder you can put in the data you downloaded from the imb server or you can put in a starter database that can be then sourced into the docker db container. 
The Base_Dir is the parent directory and I used the .. to give the implementer the choice of defining their parent directory

Reply:
@jmbabazi - I think what would be helpful would be if you can lay out some documentation (maybe in a comment at the top of the script, or in the README of the repo) that describes what the target directory structure will look like, and what is user-configurable vs. what is not.  I'd rather review the design plan for how you are intending to lay things out than the nuances of how the code is written, assuming it ultimately will achieve those goals.